src/app/common/services/project-service.coffee

```diff
@@ -214,14 +216,18 @@ angular.module("doubtfire.common.services.projects", [])
           onFailure?(failure)
       )
 
+  projectService.getGroup = (project, onSuccess) ->
+    Project.get { id: project.project_id }, (response) ->
+      project.groups = response.groups
+      onSuccess?(project)
+
   projectService.updateGroups = (project) ->
     if project.groups?
```

Comment:
Remove this if statement, and remove `getGroup` function.

Reply:
🥂